.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -8,7 +8,7 @@ body:
     attributes:
       value: |
         ### ⚠️ Warning
-        This is a public repository, so **please do not include any company-internal code or details in your bug report**.
+        This is a public repository, so **please do not include any any company-internal code or sensitive data in your bug report**.
```

Comment:
```suggestion
        This is a public repository, so **please do not include any company-internal code or sensitive data in your bug report**.
```

Reply:
@james-nash another one, i guess to be fair, we really don't want any any company internal code